src/framework/theme/styles/_theming.scss

```diff
@@ -13,7 +13,7 @@
 @import 'core/functions';
 
 $nb-enabled-themes: () !global;
-$nb-enable-css-variables: false !global;
+$nb-enable-css-variables: true !global !default;
```

Comment:
Also maybe we should use `css-custom-properties` instead of `variables`. Thats how it called in [spec](https://www.w3.org/TR/css-variables-1/#custom-property). (Please, don't look at the url 🙂)

https://stackoverflow.com/questions/48873595/is-there-a-difference-between-css-custom-properties-and-css-variables